src/Model/ValueObject/String/Behaviour/Exception/MaxLengthExceeded.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\ValueObject\String\Behaviour\Exception;
+
+use InvalidArgumentException;
+
+class MaxLengthExceeded extends InvalidArgumentException
+{
+    private function __construct(string $message)
+    {
+        parent::__construct($message);
+    }
+
+    public static function maxLengthExceeded(int $maxLength): self
+    {
+        return new self(sprintf('Given string should not be longer than %d characters.', $maxLength));
+    }
+
+    public function getPath(): string
+    {
+        return '/0/title';
```

Comment:
This makes the assumption that this trait is used inside the `Title` value object, but it is possible to use it also in the `CopyrightHolder` so then the path would be wrong.

I would not worry too much about the "correct" exposure of this error to the API layer, that will be done with the JSON schema rules.